geospatial/src/Dem.cc

```diff
@@ -110,21 +134,32 @@ int Dem::Load(const std::string &_filename)
 
   this->dataPtr->filename = fullName;
 
-  if (!exists(findFilePath(fullName)))
+  // Create a re-usable lambda for opening a dataset.
+  auto OpenInGdal = [this](const std::string& name) -> bool
```

Comment:
nit: we normally treat lambdas as local variables, so first letter should be lower case.
```suggestion
  auto openInGdal = [this](const std::string& name) -> bool
```